modyn/storage/src/CMakeLists.txt

```diff
@@ -68,51 +68,10 @@ message(STATUS "Proto generated files in ${PROTO_BINARY_DIR}: ${PROTO_GENERATED_
 
 target_include_directories(modyn-storage-proto PUBLIC "$<BUILD_INTERFACE:${PROTO_BINARY_DIR}>")
 
-target_link_libraries(modyn-storage-proto PUBLIC grpc++ grpc++_reflection)
```

Comment:
@MaxiBoether Regarding your comment on the aliasing: I just removed the alias again and it seems to work out fine :) Prior to now I just haven't tested the full matrix of options

Reply:
Ok sure, as long as it works haha